.github/workflows/update_ci.yml

```diff
@@ -95,6 +95,14 @@ jobs:
           cd plugin
           git config user.email "ci@discourse"
           git config user.name "Discourse CI"
+          git checkout -b update-ci
           git add -A
           git commit -m "DEV: Update CI workflows"
-          git push
```

Comment:
Oh, and sooner or later we'll probably hit an issue where there's already a `update-ci` branch (e.g. the previous one isn't deleted) so that's another thing we might wanna handle 🙂

Reply:
Oh man haha. I just missed typing that line in. Thanks.